input/DOI_Release_20210216_from_reserve_utf-8-bom.json

```diff
@@ -0,0 +1,46 @@
+﻿[
```

Comment:
Interesting; GitHub's UI doesn't show the BOM.

But pulling this branch and examining the file, I can indeed see it.